ppr-ui/src/store/store.ts

```diff
@@ -70,7 +79,15 @@ export const useStore = defineStore('assetsStore', () => {
   })
   /** Whether the user has 'staff' keycloak role. */
   const isRoleStaff = computed((): boolean => {
-    return (state.value.authorization?.authRoles.includes('staff') || isRoleStaffSbc.value)
+    return (state.value.authorization?.authRoles.includes(AuthRoles.STAFF) || isRoleStaffSbc.value)
+  })
+  /** Whether the user has 'manufactuer' keycloak role. */
+  const isRoleManufactuer = computed((): boolean => {
+    return (state.value.authorization?.authRoles.includes(AuthRoles.MHR_REGISTER) &&
+    state.value.authorization?.authRoles.includes(AuthRoles.MHR_PAYMENT) &&
+    state.value.authorization?.authRoles.includes(AuthRoles.MHR_TRANSPORT) &&
+    state.value.authorization?.authRoles.includes(AuthRoles.MHR_TRANSFER_SALE) &&
+    !state.value.authorization?.authRoles.includes(AuthRoles.PPR_STAFF))
   })
```

Comment:
Identifies Manufacturer based on:

![image](https://github.com/bcgov/ppr/assets/77707952/d967ed0d-aed5-4b62-bcb5-2d24cf2ea32a)

Reply:
This is good! 
Originally i was concerned about overlap with Qualified Suppliers (they are identified with `mhr_transfer_sale` role.

That being said, that role does grant access to Transfers etc, so we may need to refine that permission. A discussion you could bring up with the team perhaps! ie are manufacturers allowed to perform transfers?

Reply:
Ill bring it up thanks.

Reply:
@orelbn is this screenshot from the chat? If yes, we should post it somewhere for everyone to see. Any requirements or rules (that come up) should be added to Figma or tickets. Thanks!

Reply:
Definitely agree, good item to highlight!